transform/mattermost-analytics/models/marts/web_app/platform/eg_performance_events.sql

```diff
@@ -0,0 +1,29 @@
+{{
```

Comment:
`_platform__models.yml` with the model definitions and any tests is missing.

Reply:
Added this file, no tests as of now. Want to get a copy of this PR up and running to work with the performance team and do a 2nd revision based on what we see